.github/workflows/sync-ceremony-to-main.yml

```diff
@@ -13,18 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This workflow is triggered when a ceremony branch is completed, and will:
-# - Create a PR that merges the completed ceremony branch to main
-# - Sync the ceremony branch to the GCS preprod bucket
+# This workflow is triggered when a ceremony branch is completed,
+# creating a PR that merges the completed ceremony branch to main
 
 name: Sync Published Ceremony Branch to Main and Preprod
 
 # Declare default permissions as none.
 permissions: {}
 
 on:
-  schedule:
```

Comment:
@kommendorkapten any reason you think this is needed? i don't see why we need to continually try to sync

Reply:
I think the idea is to open a PR automatically when the ceremony branch is ready, but I think that's quite unnecessary, it's trivial to open that PR manually so I'm in favour of dropping this.

Reply:
I think this is already covered under the `push` condition, once the ceremony branch is created and pushed to.